6.X/6.9.0/docs/09-natsJs-pvc-size-increase.md

```diff
@@ -0,0 +1,423 @@
+<!-- Space: TOOLS -->
+<!-- Parent: Installer -->
+<!-- Title: NatsJs PVC size increase for 6.7.0 release-->
+<!-- Layout: plain -->
+
+<br />
+
+<!-- Include: ac:toc -->
+
+<br />
+
+## Increase persistent storage for NATS JetStream statefulset to 100Gi before upgrading on-prem to release 6.7.0
```

Comment:
This looks like a prerequisite. Kindly change to a bullet point. If you want this as a heading make it simpler and add a description